meinberlin/apps/dashboard2/mixins.py

```diff
@@ -51,19 +52,8 @@ def blueprint_key(self):
         return self.kwargs['blueprint_slug']
 
 
-class DashboardComponentMixin(base.ContextMixin,
-                              base.View):
-
-    def dispatch(self, request, project, module, *args, **kwargs):
-        self.module = module
-        self.project = project
-        return super().dispatch(request, *args, **kwargs)
-
-    def get_context_data(self, **kwargs):
-        context = super().get_context_data(**kwargs)
-        context['project'] = self.project
-        context['module'] = self.module
-        return context
+class DashboardComponentMixin(ProjectContextDispatcher):
+    component = None
```

Comment:
otherwise we would have to copy n paste the whole logic. ProjectContext is a very general function which does try to find project and module out of many possible configurations.